Sources/firebase/include/FirebaseFirestore.hh

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef firebase_include_FirebaseFirestore_hh
+#define firebase_include_FirebaseFirestore_hh
+
+#include <firebase/firestore.h>
+
+using namespace ::firebase::firestore;
```

Comment:
I think that we should avoid the using namespace.  This pollutes all users of the headers (the using is propagated to all that include the header).  Yes it makes the inline members uglier, but that is the idea - we don't want inline functions, but making these out of line is expensive because it will not get inlined and folded away on the Swift side otherwise.